extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -868,6 +874,9 @@ export class Bind {
     switch (property) {
       case 'text':
         element.textContent = String(newValue);
+        if (tag === 'TITLE') {
+          this.win_.document.title = String(newValue);
```

Comment:
`this.localWin_` is important for the "many AMPs per page via shadow DOM" PWA case.